pkg/agent/cniserver/ipam/ipam_service.go

```diff
@@ -38,7 +38,7 @@ var ipamResults = sync.Map{}
 
 type IPAMResult struct {
 	current.Result
-	VLANID uint16
+	VLANID int32
```

Comment:
I am not sure this change is needed. Just because we are using `int32` in the API shouldn't mean that we cannot use `uint16` here internally. Is there a reason for this change that I am missing?

Reply:
No other reasons. I thought we should keep them persistent in some interface code. But it is true that we still need to convert int32 to uint16 at last since vlanId seems to be of type uint16 in libopenflow. I will change them back as suggested.